tensorboard/plugins/interactive_inference/witwidget/notebook/base.py

```diff
@@ -122,7 +123,11 @@ def set_examples(self, examples):
     builder during construction. This method can change which examples WIT
     displays.
     """
-    self.examples = [json_format.MessageToJson(ex) for ex in examples]
+    if self.config.get('uses_json_input'):
+      tf_examples = self._json_to_tf_examples(examples)
```

Comment:
is there any reason we use a separate variable tf_examples for casting to tf_examples?